internal/revision/database/revision.go

```diff
@@ -29,6 +39,10 @@ import (
 	"github.com/jackc/pgx/v4"
 )
 
+var (
+	zero = []byte{}
```

Comment:
Do we still need this is the field is nullable now?

Reply:
yeah, made it nil.